bedrock/stories/templates/stories/articles/art-of-engagement.html

```diff
@@ -0,0 +1,148 @@
+      image_class="article-width center",
+    )}}
+
+    <p>“Connect is a space where the community is not dependent on our moderators, but where members can have conversations,” Siddoway said.“Our users are passionate, and when they come to the community they see that many others share their interests and also feel strongly about Firefox and Mozilla.” Case in point: Asa Dotzler, product manager for Firefox accessibility, who was a graduate student in architecture with no coding skills when he became fascinated by <a href="{{ url('stories.articles.community-champion') }}">Mozilla’s vision and joined its community.</a></p>
+  </div>
+</section>
+
+<section class="c-stories-article-section">
+  <div class="stories-max-width-wrapper">
+    <h3 class="c-stories-article-section-title">A Sense of Belonging</h3>
+    <p>Brands have been talking about community for decades, spurred largely by the rise of online forums where a single customer’s experience, for better or worse, with a company or product better or worse, could be broadcast to far more people than only their family, friends and neighbors. David W. McMillan and David M. Chavis, professors at Vanderbilt University, defined community in 1986 as a feeling that members have of belonging, where members matter to each other and have faith that their needs will be met. Since then, a growing body of academic research and business books has delved into refining that notion and determining what makes a brand community succeed.</p>
+    <p>Even so, community remains a vague and slippery concept at many companies. Often it is just another word marketers use to describe customers, and frequently it is a one-way street down which brands encourage consumers to take on some of the work of spreading the word about their products. That’s never been the case at Mozilla, where a passionate, highly engaged volunteer community has helped build its products from the early, bug-filled days of the first Firefox browser.</p>
+
+    {{ stories_image(
+      image_url="img/stories/engagement-hero.png",
+      include_highres=True,
+      alt="illustration of a brain in a jar",
+      width="1010",
+      height="519",
+      loading="lazy",
+      image_class="article-width center",
+      caption="Image by: Opensource.com / CC BY-SA 4.0"
+    )}}
+
+    <p>At the core of the 1998 Mozilla project was a commitment to open-source development, (Link to License story) and the volunteers who believed in that model played a vital role in developing Firefox 1.0, released six years later. Since then, thousands of people have offered ideas, suggestions and criticisms that continue to shape the browser and other Mozilla products. Between 2017 and 2019 alone, 22,821 non-employees made more than 2 million contributions to Mozilla products, according to the 2020 report “Mozilla and the Rebel Alliance.” Almost 750,000 of those contributions focused on Firefox, with more than 65,000 specifically involving code.</p>
```

Comment:
Missing link:
```suggestion
    <p>At the core of the 1998 Mozilla project was a <a href="{{ url('stories.build-together') }}">commitment to open-source development</a>, and the volunteers who believed in that model played a vital role in developing Firefox 1.0, released six years later. Since then, thousands of people have offered ideas, suggestions and criticisms that continue to shape the browser and other Mozilla products. Between 2017 and 2019 alone, 22,821 non-employees made more than 2 million contributions to Mozilla products, according to the 2020 report “Mozilla and the Rebel Alliance.” Almost 750,000 of those contributions focused on Firefox, with more than 65,000 specifically involving code.</p>
```